src/vivarium_gates_mncnh/constants/data_values.py

```diff
@@ -63,6 +63,12 @@ class _Durations(NamedTuple):
     "Pakistan": 0.514583,
 }
 
+ANC_ULTRASOUND = {"Ethiopia": 0.607, "Nigeria": 0.587, "Pakistan": 0.667}
+
+ANC_AI_ULTRASOUND = {"Ethiopia": 0, "Nigeria": 0, "Pakistan": 0}
+
+STATED_GA_AT_ANC = {"no_ultrasound": 45.5, "standard_ultrasound": 20, "ai_ultrasound": 5}
+
```

Comment:
Nit: I would make these names a little more informative and store them in a NamedTuple class. Note: THis is fine because Im going to overwrite this soon anyways so you do not need to change it this time.

Reply:
On STATED_GA_AT_ANC you should divide all the numbers by 7 since that is in days and gestational age is in weeks.